templates/default/matterhorn-harness.erb

```diff
@@ -145,7 +145,19 @@ NEWRELIC="-javaagent:${NEW_RELIC_JAR} $NEWRELIC_CONFIG"
 NEWRELIC=""
 <% end %>
 
-MATTERHORN_OPTS="$NEWRELIC $GOSH $DEBUG $FELIX $GRAPHICS $JAVA $LOGGING $JMX $DISABLE_PHONEHOME $ADD_OPTS"
+<% if @enable_yourkit_agent %>
+YOURKIT="-agentpath:<%= @yourkit_dir %>/libyjpagent.so=dir=<%= @yourkit_dir %>,onlylocal"
```

Comment:
fyi, the `dir=..l` arg in that agentpath string tells yourkit where memory snapshots should be created.